### ISSUE TITLE
Can't connect to webchat integration on local host after updating to new version of Rasa 1.9.3

### ISSUE THREAD

**Issue by Jainu-s** [RasaHQ/rasa#5539]
**Currently I'm using** 
Rasa version: 1.9.3
Rasa X version: 0.27.2
Rasa SDK version: 1.9.0

when I run `" rasa run -m models --enable-api --cors “*” --debug "` to connect with webchat local host I cannot see anything on index page. In the early versions I don't have this kind of errors but when I installed Rasa 1.9.3 and Rasa x 0.27.1 I'm getting this kind of errors. Even after updating to Rasa X 0.27.2 I can't resolve this issue and I raised question in forum but none have replied to it.

When I run the command " rasa run -m models --enable-api --cors “*” --debug " it wasn't thrown any error but showing like this:

```
(venv) (base) user@DELL-Admin:~/PycharmProjects/Rasa_chatbot$ rasa run -m models --enable-api --cors “*” --debug
2020-03-31 16:02:30 DEBUG    rasa.cli.utils  - Parameter 'endpoints' not set. Using default location 'endpoints.yml' instead.
2020-03-31 16:02:30 DEBUG    rasa.cli.utils  - Parameter 'credentials' not set. Using default location 'credentials.yml' instead.
2020-03-31 16:02:33 DEBUG    rasa.core.utils  - Available web server routes: 
/conversations/<conversation_id>/messages          POST                           add_message
/conversations/<conversation_id>/tracker/events    POST                           append_events
/webhooks/rasa                                     GET                            custom_webhook_RasaChatInput.health
/webhooks/rasa/webhook                             POST                           custom_webhook_RasaChatInput.receive
/webhooks/rest                                     GET                            custom_webhook_RestInput.health
/webhooks/rest/webhook                             POST                           custom_webhook_RestInput.receive
/model/test/intents                                POST                           evaluate_intents
/model/test/stories                                POST                           evaluate_stories
/conversations/<conversation_id>/execute           POST                           execute_action
/domain                                            GET                            get_domain
/socket.io                                         POST                           handle_request
/                                                  GET                            hello
/model                                             PUT                            load_model
/model/parse                                       POST                           parse
/conversations/<conversation_id>/predict           POST                           predict
/conversations/<conversation_id>/tracker/events    PUT                            replace_events
/conversations/<conversation_id>/story             GET                            retrieve_story
/conversations/<conversation_id>/tracker           GET                            retrieve_tracker
/webhooks/socketio                                 GET                            socketio_webhook.health
/status                                            GET                            status
/model/predict                                     POST                           tracker_predict
/model/train                                       POST                           train
/conversations/<conversation_id>/trigger_intent    POST                           trigger_intent
/model                                             DELETE                         unload_model
/version                                           GET                            version
2020-03-31 16:02:33 INFO     root  - Starting Rasa server on http://localhost:5005
2020-03-31 16:02:33 DEBUG    rasa.core.utils  - Using the default number of Sanic workers (1).
2020-03-31 16:02:33 INFO     root  - Enabling coroutine debugging. Loop id 93960808.
2020-03-31 16:02:33 DEBUG    rasa.model  - Extracted model to '/tmp/tmp29a2ixme'.
2020-03-31 16:02:36 DEBUG    rasa.utils.tensorflow.models  - Loading the model ...
2020-03-31 16:02:36.087698: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-03-31 16:02:40 DEBUG    rasa.utils.tensorflow.models  - Finished loading the model.
2020-03-31 16:02:40 DEBUG    rasa.utils.tensorflow.models  - Building tensorflow prediction graph...
2020-03-31 16:02:46 DEBUG    rasa.utils.tensorflow.models  - Finished building tensorflow prediction graph.
2020-03-31 16:02:46 DEBUG    rasa.nlu.classifiers.diet_classifier  - Failed to load model. Maybe the path '/tmp/tmp29a2ixme/nlu' doesn't exist?
2020-03-31 16:02:46 DEBUG    rasa.core.tracker_store  - Connected to InMemoryTrackerStore.
2020-03-31 16:02:46 DEBUG    rasa.core.lock_store  - Connected to lock store 'InMemoryLockStore'.
2020-03-31 16:02:46 DEBUG    rasa.model  - Extracted model to '/tmp/tmpdxdi2op5'.
2020-03-31 16:02:46 DEBUG    pykwalify.compat  - Using yaml library: /home/user/PycharmProjects/Rasa_chatbot/venv/lib/python3.6/site-packages/ruamel/yaml/__init__.py
2020-03-31 16:02:47 DEBUG    rasa.utils.tensorflow.models  - Loading the model ...
2020-03-31 16:02:47 DEBUG    rasa.utils.tensorflow.models  - Finished loading the model.
2020-03-31 16:02:47 DEBUG    rasa.utils.tensorflow.models  - Building tensorflow prediction graph...
2020-03-31 16:02:48 DEBUG    rasa.utils.tensorflow.models  - Finished building tensorflow prediction graph.
2020-03-31 16:02:48 DEBUG    rasa.core.nlg.generator  - Instantiated NLG to 'TemplatedNaturalLanguageGenerator'.
2020-03-31 16:03:27 DEBUG    rasa.core.channels.socketio  - User f983527143cd4d25be041da028010aea connected to socketIO endpoint.
2020-03-31 16:03:28 DEBUG    rasa.core.channels.socketio  - User aacbb309ca114db5b434e15f4e0f7318 connected to socketIO endpoint.
2020-03-31 16:03:29 DEBUG    rasa.core.channels.socketio  - User b74a2aea053540a288998ee28416a888 connected to socketIO endpoint.
2020-03-31 16:03:32 DEBUG    rasa.core.channels.socketio  - User 87a082aebdfb45c9a2206e68ad74777a connected to socketIO endpoint.
2020-03-31 16:03:35 DEBUG    rasa.core.channels.socketio  - User e71ff9b8cf554ab9bafdde074036b1ce connected to socketIO endpoint.
2020-03-31 16:03:36 DEBUG    rasa.core.channels.socketio  - User fcb246967fe24e9e9e39168c4fe6e5a0 connected to socketIO endpoint.
2020-03-31 16:03:37 DEBUG    rasa.core.channels.socketio  - User b55b1ee92dc443fb9576c9f054f24c03 connected to socketIO endpoint.
2020-03-31 16:03:39 DEBUG    rasa.core.channels.socketio  - User c7b1870abe314912a0eb4b9e0572a600 connected to socketIO endpoint.
2020-03-31 16:03:42 DEBUG    rasa.core.channels.socketio  - User 4bc96c79847c4ce7b5004c47ec8d4bda connected to socketIO endpoint.
2020-03-31 16:03:47 DEBUG    rasa.core.channels.socketio  - User 94fe32a67f264be4b9aeb0dc233cd45c connected to socketIO endpoint.
2020-03-31 16:03:52 DEBUG    rasa.core.channels.socketio  - User 7779dc25a90e4324b003f2459f14e214 connected to socketIO endpoint.
2020-03-31 16:03:57 DEBUG    rasa.core.channels.socketio  - User 80b1f82faf1c43a69720f715b0a207c9 connected to socketIO endpoint.
2020-03-31 16:13:26 DEBUG    rasa.core.channels.socketio  - User 2ba3838380764ee6bf574459ba573543 disconnected from socketIO endpoint.
2020-03-31 16:13:27 DEBUG    rasa.core.channels.socketio  - User a6ef81d7d07e481c9f8570d6aba627b2 disconnected from socketIO endpoint.
2020-03-31 16:13:27 DEBUG    rasa.core.channels.socketio  - User ab3c02480c634dc4bf7e3c9b8139b7c0 disconnected from socketIO endpoint.
2020-03-31 16:13:28 DEBUG    rasa.core.channels.socketio  - User d29951b9413a45659bb56114e9845a1d disconnected from socketIO endpoint.


```

**Here is my credentials.yml file**

```
# This file contains the credentials for the voice & chat platforms
# which your bot is using.
# https://rasa.com/docs/rasa/user-guide/messaging-and-voice-channels/

rest:
#  # you don't need to provide anything here - this channel doesn't
#  # require any credentials


#facebook:
#  verify: "<verify>"
#  secret: "<your secret>"
#  page-access-token: "<your page access token>"

#slack:
#  slack_token: "<your slack token>"
#  slack_channel: "<the slack channel>"

socketio:
  user_message_evt: user_uttered
  bot_message_evt: bot_uttered
  session_persistence: true

#mattermost:
#  url: "https://<mattermost instance>/api/v4"
#  token: "<bot token>"
#  webhook_url: "<callback URL>"

# This entry is needed if you are using Rasa X. The entry represents credentials 
# for the Rasa X "channel", i.e. Talk to your bot and Share with guest testers.
rasa:
  url: "http://localhost:5002/api"

```
**Here is my endpoint.yml file**

```
# This file contains the different endpoints your bot can use.

# Server where the models are pulled from.
# https://rasa.com/docs/rasa/user-guide/configuring-http-api/#fetching-models-from-a-server/

#models:
#  url: http://my-server.com/models/default_core@latest
#  wait_time_between_pulls:  10   # [optional](default: 100)

# Server which runs your custom actions.
# https://rasa.com/docs/rasa/core/actions/#custom-actions/

action_endpoint:
  url: "http://localhost:5055/webhook"

# Tracker store which is used to store the conversations.
# By default the conversations are stored in memory.
# https://rasa.com/docs/rasa/api/tracker-stores/

#tracker_store:
#    type: redis
#    url: <host of the redis instance, e.g. localhost>
#    port: <port of your redis instance, usually 6379>
#    db: <number of your database within redis, e.g. 0>
#    password: <password used for authentication>
#    use_ssl: <whether or not the communication is encrypted, default false>

#tracker_store:
#    type: mongod
#    url: <url to your mongo instance, e.g. mongodb://localhost:27017>
#    db: <name of the db within your mongo instance, e.g. rasa>
#    username: <username used for authentication>
#    password: <password used for authentication>

# Event broker which all conversation events should be streamed to.
# https://rasa.com/docs/rasa/api/event-brokers/

#event_broker:
#  url: localhost
#  username: username
#  password: password
#  queue: queue

```

 **Here is my html source:**

```
<!DOCTYPE html>
<html>
 
<head>
<title>Chatbot</title>
</head>
  
<body>
  
<div id="webchat" />
<script src="https://storage.googleapis.com/mrbot-cdn/webchat-latest.js"></script>
// Or you can replace latest with a specific version
<script>
WebChat.default.init({
selector: "#webchat",
initPayload: "/get_started",
customData: { "language": "en" }, // arbitrary custom data. Stay minimal as this will be added to the socket
 socketUrl: "http://localhost:5005",//add the ip address of the system where your rasa bot is running
 socketPath: "/socket.io/",
 title: "Title",
 subtitle: "Subtitle",
 })
 </script>
</body>
</html>
```
**I have no active functions in my actions.py file**

**Comment by sara-tagger** [comment]
Thanks for the issue, @erohmensing will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by erohmensing** [comment]
Hi there, which version was it working on? What do the network requests look like on the index page (e.g. inspect network requests using chrome developer tools)?

**Comment by erohmensing** [comment]
Also a very supid question -- in the command you attached, it filled it with `“*”` instead of `"*"` -- can you make sure that you're launching with the latter?

**Comment by Jainu-s** [comment]
> Also a very supid question -- in the command you attached, it filled it with `“*”` instead of `"*"` -- can you make sure that you're launching with the latter?

It's my mistake actually as I copied it from rasa forum and nothing wrong with the version. I updated my version and I got rasa x error with frontend dependencies error and I thought this is also an bug in it. Finally it works thank you @erohmensing  I can see you everywhere on Rasa HQ Github.

**Comment by erohmensing** [comment]
No problem! Stuff like that is difficult to catch sometimes!

**Comment by erohmensing** [comment]
PS @Jainu-s, the Rasa X frontend dependencies should have been fixed in 0.27.2, but I think you may have figured that out already 🙂
